exec/java-exec/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -111,6 +111,65 @@
       },
       "enabled" : false
     },
+    "os" : {
```

Comment:
@vdiravka 
Thanks for this PR.  I'm concerned with calling this config `os` as that kind of implies `operating system` and there are similar tools that query operating systems.  In the example config, could we perhaps call it `oci` or `oracle_cloud` or something besides `os`?

Reply:
agree. I also named that `oci` when tested locally. The reason why I named it OS, because it corresponds to `S3` in Amazon:
`aws` = `oci` and `s3` = `os`.
But since we don't expect any other Storage Plugin for Oracle Cloud Infrastructure, we can name it as `oci`.